### PR TITLE
Support to import Gitlab repository

### PR DESCRIPTION
**Pull request by studioego**
Support to import Gitlab repository.
add default repository input box (ex. main, master, core, etc) for import Gitlab repository
add gitlab exception messages, change exception message notification location.
add i18n messages for Gitlab messages
https://github.com/lablup/backend.ai-webui/issues/1267

### CODE REVIEW

File: resources/i18n/en.json [hunk 1]
```diff
@@ -1119,7 +1119,11 @@
     "Installed": "Installed",
     "GithubInternalError": "Github Internal Error.",
     "GithubAPILimitError": "Github API Limitation Error.",
-    "GithubAPIEtcError": "Github Server Issue."
+    "GithubAPIEtcError": "Github Server Issue.",
+    "ImportGitlabRepo": "Import Gitlab Repository",
+    "GitlabRepoWillBeFolder": "Imported GitLab repositories are converted to folders and can be mounted when session starts.",
+    "GitLabURL": "GitLab URL",
+    "GitLabDefaultBranch": "Gitlab Branch Name"
```
Comment (adrysn) [inline resources/i18n/en.json]: Let's be consistent with the naming for the messages: `Gitlab` or `GitLab`. Please choose one.

File: src/components/backend-ai-import-view.ts [hunk 2]
```diff
@@ -279,6 +280,50 @@ export default class BackendAIImport extends BackendAIPage {
     }
   }
 
+  getGitLabRepoFromURL() {
+    let url = this.shadowRoot.querySelector('#gitlab-repo-url').value;
+    let tree = 'master';
+    let get_branch_name = this.shadowRoot.querySelector('#gitlab-default-branch-name').value;
```
Comment (adrysn) [inline src/components/backend-ai-import-view.ts]: Let's use the camel-case variable name if there is no other reason. The syntax should automatically be checked with some linting tools, but it is disabled for now.

Comment (adrysn) [inline src/components/backend-ai-import-view.ts]: Let's use the camel-case variable name if there is no other reason. The syntax should automatically be checked with some linting tools, but it is disabled for now.

**Review by adrysn — APPROVED** [review]
Minor comments on the naming conventions. The feature is working fine.

File: resources/i18n/en.json (hunk 1 above)
Comment (studioego) [reply resources/i18n/en.json]: I choose `Gitlab`.

File: src/components/backend-ai-import-view.ts (hunk 2 above)
Comment (studioego) [reply src/components/backend-ai-import-view.ts]: I'll change it to the camel-case variable name.

Comment (studioego) [reply src/components/backend-ai-import-view.ts]: change `get_branch_name` to `getBranchName` according to camel-case.

**Pull request merged**